src/js/ext-win-script.js

```diff
@@ -7,29 +7,39 @@
  */
 /* eslint-disable */
 (function($, Drupal) {
-  var $extLinkText = Drupal.t('external link opens in a new window / tab'),
-    $intLinkText = Drupal.t('opens in a new window / tab');
-  // Internal links with data-ext-url - turn them into external links
-  $("#main-content a[data-ext-type^='External']").each(function() {
-    $(this).attr('href', $(this).attr('data-ext-url'));
-    if ($(this).parent().hasClass('moreLink')) {
-      $(this).html('visit site');
+  Drupal.behaviors.nicsdruOriginsExtWin = {
+    attach: function (context) {
+      var $extLinkText = Drupal.t('external link opens in a new window / tab'),
+        $intLinkText = Drupal.t('opens in a new window / tab');
+
+      // External links - add identifiers
+      $("#main-content a[href*='http://'],#main-content a[href*='https://'],#main-content a[href^='//']", context)
```

Comment:
More nit picking :)

Separate parameters with a space after the comma for improved readability.